AI-and-Analytics/Getting-Started-Samples/IntelPython_XGBoost_GettingStarted/README.md

```diff
@@ -22,6 +22,8 @@ In this code sample, you will learn how to use Intel optimizations for XGBoost p
 | Hardware                | Intel Atom® Processors <br> Intel® Core™ Processor Family <br> Intel® Xeon® Processor Family <br> Intel® Xeon® Scalable processor family
 | Software                | XGBoost* 
 
+> **Note**: AI and Analytics samples are validated on AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).
+
 ## Key Implementation Details
 
 - This Getting Started sample code is implemented for CPU using the Python language. The example assumes you have XGboost installed inside a conda environment, similar to what is delivered with the installation of the Intel® Distribution for Python* as part of the [Intel® AI Tools](https://software.intel.com/en-us/oneapi/ai-kit).
```

Comment:
Stefana @raistefintel, please take a look at the latest template version: https://github.com/oneapi-src/oneAPI-samples/wiki/AI-Getting-Started-and-Features-and-Functionality-README-Template (updated on April 19th), we made several changes to improve the template based on the UAT results. For example, in the new version Env setup section has been extended with a few more steps. I don't see it here and I can't suggest changes to this part of the file (it's grey, I don't have + option to leave a comment)